tests/Transaction/Manager-test.cpp

```diff
@@ -95,12 +95,12 @@ class TransactionManagerTest : public ::testing::Test {
 
 TEST_F(TransactionManagerTest, parsing_errors) {
   auto json = arangodb::velocypack::Parser::fromJson("{ \"write\": [33] }");
-  Result res = mgr->createManagedTrx(vocbase, tid, json->slice());
```

Comment:
Can we add tests with `true` for follower Transaction and assert that the HINT is set, and otherwise behavior is identical.
Maybe we can run this suite in a templated Test variant for true/false ?

Reply:
Done.